src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -31,8 +31,6 @@ const Body = BaseForm.extend(Object.assign(
               type: 'error',
             };
             if (this.options.appState.containsMessageWithI18nKey(OV_NMC_FORCE_UPGRAGE_SERVER_KEY)) {
-              // account for error customization
```

Comment:
This is something I am fixing from my previous PR. We just need to add the server returned i18n key to login.properties to account for i18n.